.circleci/config.yml

```diff
@@ -23,65 +14,41 @@ jobs:
       - checkout
       - restore_cache:
           key: dependency-cache-{{ checksum "package.json" }}
-      - run:
-          name: Install dependencies
-          command: npm i
+      - run: npm i
       - save_cache:
           key: dependency-cache-{{ checksum "package.json" }}
           paths:
             - ./node_modules
-  testnode12:
-    <<: *defaults
-    docker:
-      - image: circleci/node:12
-    steps:
-      - runtests
-  testnode14:
-    <<: *defaults
-    docker:
-      - image: circleci/node:14
-    steps:
-      - runtests
-  testnode16:
-    <<: *defaults
-    docker:
-      - image: circleci/node:16
-    steps:
-      - runtests
-  testnode18:
-    <<: *defaults
-    docker:
-      - image: cimg/node:18.17.1
-    steps:
-      - runtests
-  testnode20:
-    <<: *defaults
-    docker:
-      - image: cimg/node:20.5.1
-    steps:
-      - runtests
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./node_modules
```

Comment:
If you persist `node_modules` to workspace I suppose you don't need to save it to cache anymore?

Reply:
The cache persists between runs and makes it quicker to run the next commit